pyproject.toml

```diff
@@ -3,10 +3,13 @@ max_line_length = 120
 
 [tool.isort]
 multi_line_output = 5
-line_length = 119
+line_length       = 119
 
 [tool.pytest.ini_options]
 testpaths = ["tests"] # Search for tests in tests/
 
 [tool.coverage.run]
-omit = ["venv/*"]  # when running pytest --cov, dont report coverage in venv directories
+omit = ["venv/*"] # when running pytest --cov, dont report coverage in venv directories
+
+[tool.pyright]
+# pythonPlatform = 'Darwin'
```

Comment:
Also leftover from testing?  There's some value to keeping it as reference of course.